staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -903,6 +903,13 @@ type Features struct {
 	Pvspinlock *FeatureState `json:"pvspinlock,omitempty"`
 }
 
+//
+// +k8s:openapi-gen=true
+type SyNICTimer struct {
+	Enabled *bool         `json:"enabled,omitempty"`
+	Direct  *FeatureState `json:"direct,omitempty"`
```

Comment:
Isn't this breaking change?

Reply:
@xpivarc in what sense?

Reply:
Sorry, I didn't checked the `FeatureState` struct. All good since we have `Enabled` here.